src/test/java/org/cyclops/integrateddynamics/core/evaluate/variable/TestStringOperators.java

```diff
@@ -792,4 +792,32 @@ public void testInvalidInputTypeNamedName() throws EvaluationException {
         Operators.NAMED_NAME.evaluate(new IVariable[]{DUMMY_VARIABLE});
     }
 
+    /**
+     * ----------------------------------- STRING_ERROR -----------------------------------
+     */
+
+    @Test(expected = EvaluationException.class)
+    public void testStringErrorWithSymbols() throws EvaluationException {
+        Operators.STRING_ERROR.evaluate(sregex);
+    }
+
+    @Test(expected = EvaluationException.class)
+    public void testStringErrorWithSpaces() throws EvaluationException {
+        Operators.STRING_ERROR.evaluate(shelloWorld);
+    }
+
+    @Test(expected = EvaluationException.class)
+    public void testInvalidInputSizeLong() throws EvaluationException {
```

Comment:
Isn't this also supposed to be called an Error in the method name? the others all have it

Reply:
yes, fixing it now